action.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Adevinta
+
+name: Lava
+description: Run Lava
+inputs:
+  version:
+    description: Lava version
+    default: latest
+  config:
+    description: Path of the Lava configuration file
+    default: lava.yaml
```

Comment:
This could be blank, as lava already has the default `lava.yaml`, and pass the `-c` just in case it's specified (and perhaps if the file exists)

Although your way is more evident the name of the config file.

Wondering if the config file would be always requiered for lava.

Reply:
> Wondering if the config file would be always required for lava.

That is a very good point. We will start working soon on defining Lava's defaults and its distribution and update system. This should be one of the outcomes of these tasks.